src/main/java/org/folio/circulation/domain/CheckInProcessRecords.java

```diff
@@ -3,7 +3,10 @@
 import java.util.Optional;
 import java.util.UUID;
 
+import org.joda.time.DateTime;
+
```

Comment:
Why is there a gap between org.joda and org.folio package namespaces?

I've noticed this on another pull request recently. Do you use the Eclipse import plugin?

Reply:
Or has this already been resolved since my previous comment and the changes haven't been pushed?

Reply:
I used eclipse code formatter plugin and folio.importorder from the jira ticket you supplied.

Reply:
I think something has changed in the configuration of the eclipse formatter recently (I don’t use it).

I believe @roman-barannyk had similar challenges on his pull request as well, see https://github.com/folio-org/mod-circulation/pull/417#discussion_r363305594

Reply:
What do you use? Do you format imports manually? Can I have your settings?

Reply:
Can you update the jira ticket about the formatting that should be used?

Reply:
I use IntelliJ to order the imports. I believe my settings are the same as the `Eclipse style imports.xml` configuration file on the issue.

I don't know what was changed recently to make the eclipse formatting plugin change behaviour.

Reply:
I used the eclipse style imports and it formats the way it is currently pushed.

Reply:
I re-imported the xml file settings from the issue into IntelliJ and ran [Organise inmports|https://stackoverflow.com/questions/8608710/intellij-organize-imports].

This changed the import order to
```
import java.util.Optional;
import java.util.UUID;

import org.folio.circulation.domain.representations.CheckInByBarcodeRequest;
import org.folio.circulation.support.ClockManager;
import org.joda.time.DateTime;
```

As I understand it, that fits with the provisionally agreed standard.

What happens if you try that?

Reply:
Done.

Reply:
Please can this also be done to the other files that have changed in this pull request, as quite a few others seem to be affected